### PR TITLE
Dao hero

### PR DESCRIPTION
**Pull request by mkubdev**
### Describe the changes you've made

Add: New homepage + Tailwind + Lottie animation

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## How Has This Been Tested?

Same as before.

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

 Original           | Updated
 :--------------------: |:--------------------:
image | image |
 
## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by KukretiShubham — APPROVED** [review]
Apart from logo, everything is OP🔥

**Review by shuklaritvik06 — APPROVED** [review]
LGTM

**Review by vinzvinci — APPROVED** [review]
Awesome! LGTM 💯

File: components/Hero.js [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+                    <span className="ml-4 text-sm">Follow us on GitHub</span>
+                    <ChevronRightIcon
+                      className="ml-2 w-5 h-5 text-gray-500"
+                      aria-hidden="true"
+                    />
+                  </a>
+                  <h1 className="mt-4 text-4xl tracking-tight font-extrabold text-white sm:mt-5 sm:leading-none lg:mt-6 lg:text-5xl xl:text-6xl">
+                    <span className="md:block">Join WebX DAO</span>{" "}
+                    <span className="text-purple-400 md:block">
+                      Opensource Community
```
Comment (vinzvinci) [inline components/Hero.js:158]: Give space, like

Open Source

Comment (mkubdev) [reply components/Hero.js:158]: Sure!

**Pull request merged**